### PR TITLE
Add location how-to

### PR DESCRIPTION
**Pull request by dnewh**
This adds a walkthrough for creating, updating, viewing, and deleting Grid Locations on a Splinter network. This is similar to the walkthrough for Grid products.

### CODE REVIEW

File: docs/0.1/creating_locations.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+     location schema name (`gs1_location` for this example).
+   * Location ID (such as a GTIN)
+   * Owner (organization ID)
+   * Set of location attributes (each with a name and value that conforms to
+     the data type)
+     <br><br>
+
+    ```yaml
+    - namespace: "GS1"
+      location_id: "9012345000004"
```
Comment (rbuysse) [inline docs/0.1/creating_locations.md:239]: `location_id` and `owner`should be updated to real values to match opened PR#168

```
  location_id: "0013600000011"
  owner: "013600"
```

File: docs/0.1/creating_locations.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+1. Use `docker cp` to copy the file into the `gridd-alpha` container.
+
+   ```
+   $ docker cp location.yaml gridd-alpha:/
+   ```
+
+1. Add the new location by using the `grid location create` command to specify the
+   definition in the `location.yaml` file.
+
+   ```
```
Comment (rbuysse) [inline docs/0.1/creating_locations.md:267]: Before creating the location, the user needs to have permissions.

```
grid \
agent update 013600 $(cat ~/.grid/keys/alpha-agent.pub) --active \
--role can_create_location \
--role can_update_location \
--role can_delete_location \
--role admin
```

**Review by rbuysse — CHANGES_REQUESTED** [review]
Oops I left the changes requested as comments instead.

File: docs/0.1/creating_locations.md (hunk 2 above)
Comment (dnewh) [reply docs/0.1/creating_locations.md:267]: User creation is covered in a different docs page and the necessary permissions are listed on line 219-220

File: docs/0.1/creating_locations.md (hunk 1 above)
Comment (dnewh) [reply docs/0.1/creating_locations.md]: updated

**Review by dplumb94 — APPROVED** [review]

File: docs/0.1/creating_locations.md [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+
+{:start="8"}
+
+8. List all existing locations to verify that the new location has been added.
+
+   ```
+   root@gridd-alpha:/# grid location list
+   Location ID: 9012345000004
+   Namespace: GS1
+   Owner: 314156
```
Comment (dplumb94) [inline docs/0.1/creating_locations.md:289]: Looks like this is not using the updated location list output (and its an old owner ID)

**Review by dplumb94 — CHANGES_REQUESTED** [review]

File: docs/0.1/creating_locations.md (hunk 3 above)
Comment (dnewh) [reply docs/0.1/creating_locations.md]: updated

**Review by rbuysse — DISMISSED** [review]

**Review by dplumb94 — APPROVED** [review]

**Review by rbuysse — APPROVED** [review]

**Pull request merged**